src/app/tests/suites/certification/Test_TC_DA_1_9.yaml

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: Add the test cases for indirect CRLs
+
+name: 29.1.9. [TC-DA-1.9] Device Attestation Revocation [DUT-Commissioner]
+
+PICS:
+    - MCORE.ROLE.COMMISSIONER
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic Information"
+    endpoint: 0
+
+tests:
+    - label: "Precondition/Test case description"
+      verification: |
+          (for all steps, PIs use equivalent command on their respective platform)
+          Validate that the DUT properly handles revoked device attestation certificates during commissioning, including certificates revoked through indirect CRLs.
+          set the $CHIP_ROOT environment variable to the location of the connectedhomeip directory (cd to the connectedhomeip directory, use command `export CHIP_ROOT=$(pwd -P)`
+          For testing Device Attestation Revocation feature, commissioners must allow commissioning the end device with Vendor Id 0xFFF1.
+          For each test data set scenario outlined in the TC-DA-1.9, perform the following actions
+          - Start the TH using the required device attestation certificate set.
+          - Have the DUT commission the TH. If either DAC or PAI certificate has been revoked, verify that the DUT warns the user that they are commissioning a non-genuine device, or otherwise indicates a failure of device attestation, within its error handling APIs or user interface. If either the DAC or PAI is not revoked, verify that the DUT successfully commissions the TH.
+          - factory reset the TH.
+      disabled: true
+
+    - label:
```

Comment:
Is it not possible to make this test into a Python test that properly configured the TH and prompts the user? I think it is possible to do so.

Reply:
It is, added it